engine/runtime/src/test/java/org/enso/compiler/ParseStdLibTest.java

```diff
@@ -13,15 +13,9 @@
 import java.util.Arrays;
```

Comment:
I think we can remove this test altogether. All the standard library sources are processed when generating "library manifests" anyway. No point in parsing them again when there is no old AST to compare the result to.